content/english/Kusto-KC7/activity-0.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Activity 0: Login & Setup of the Cyber Environment"
+description: ""
+draft: false
+weight: 4
+---
+
+{{< youtube VL7fJr54CG0 >}}
+
+# Getting Started
+
+Okay Cyber Defenders! The first thing you need to do is set up the environment in which you will be hunting hackers. In order to do that, open the links below and follow the directions on first setting up Azure Data Explorer (ADX) and then logging into the Scoreboard session.
+- Azure Data Explorer : https://dataexplorer.azure.com/ 
+- The Scoreboard : https://aka.ms/kc7scoreboard 
+- (Optional) PDF Guide of This Workshop : https://kc7cyber.com/guide1 
+
+
+
+<!-- ![resource](../kusto-kc7/Images/Resources.png) -->
+<img src= "https://github.com/bgrant34/workshops/blob/master/content/english/kusto-kc7/Images/Resources.png?raw=true" alt= “” width="80%" height="value">
+
+### Setting Up Azure Data Explorer (ADX)
+
+ADX is the primary tool used by some cyber defenders for data exploration and analysis. The great thing about ADX is that it is used by cyber analysts at many of the smallest and largest organizations in the world. 
+
+Let’s get you logged in and started with ADX:
+
+1.	Go to https://dataexplorer.azure.com/ and login with your Microsoft account 
```

Comment:
We might want to indicate as a pre-requisite that you must have a Microsoft account